std_msgs/Header header
BoundaryCondition[] bound_conds